engine/include/cubos/engine/voxels/palette_bridge.hpp

```diff
@@ -0,0 +1,27 @@
+/// @file
+/// @brief Class @ref cubos::engine::PaletteBridge.
+/// @ingroup voxels-plugin
+
+#pragma once
+
+#include <cubos/engine/assets/bridges/file.hpp>
+
+namespace cubos::engine
+{
+    /// @brief Bridge which loads and saves @ref Palette assets.
+    ///
+    /// Palettes are composed by n materials.
+    /// They're stored as an array of floats (each 4 floats equal a material).
```

Comment:
Here you can just refer that it uses the same format as `@ref VoxelPalette::loadFrom and @ref VoxelPalette::writeTo`.